public/css/widget/view-mode-switcher.less

```diff
@@ -9,9 +9,10 @@
   }
 
   label {
-    color: @icinga-blue;
+    display: inline-block;
```

Comment:
What is this for?

Reply:
This is a change from @nilmerg. Can you please explain?

Reply:
Good question...

Reply:
Can't remember why I've added this. Doesn't seem to have an effect anymore either. Can be removed.